docs/welcome.md

```diff
@@ -1,54 +1,59 @@
-- **[Import Your Existing DB](/how-to/import-prisma-schema/)**: Use your _existing_ databases for a head start on development.
-- **[Full Stack Code Generation](/getting-started/)**: Fully-functional REST and GraphQL APIs, database models, authentication, logging, React Admin UI, and more.
-- **[Event-Driven Architecture](/concepts/event-driven-architecture/)**: Create new services with a modern architecture or modernize your existing apps into a event-driven microservices architecture.
-- **[Seamless Git Integration](/smart-git-sync/)**: Smart git sync with GitHub, Bitbucket, GitLab, or AWS CodeCommit.
-- **[Custom Code Integration](/custom-code/)**: Extend generated code with your own custom business logic.
-- **[Plugin System](/getting-started/plugins/)**: Extend functionality with our community plugins or create your custom plugins.
-- **[Multiple Databases](/getting-started/#databases)**: Support for PostgreSQL, MySQL, MongoDB, MS SQL Server, and more.
-- **Deployment & CI/CD**: Simplify and automate your CI/CD pipeline with our plugins and streamline deployments to popular options like [AWS ECS](/deploy/aws/ecs), [Docker Desktop](/deploy/docker-desktop/), and [Kubernetes](/deploy/kubernetes).
+- **Live Service Templates**
+- **[Private Plugins](/getting-started/plugins/)**: Extend functionality with private plugins or use our community plugins.
+- **Platform Console**
+
+## Technical Debt Prevention
 
-## Tutorials and Samples
+Track and manage technical debt through our Platform Console and Service Catalog.
+Monitor template versions, plugin updates, and dependency changes across all services.
+Receive automatic notifications when updates are available and get clear upgrade paths for outdated services.
+Maintain consistent standards across your entire service portfolio.
 
-- Follow our [step-by-step tutorials](/tutorials/) to build real-world applications.
-- Explore our [sample applications](https://github.com/amplication/sample-app) on GitHub.
+- Technical Debt Management
+- Service Catalog
+- **[Seamless Git Integration](/smart-git-sync/)**: Smart git sync with GitHub, Bitbucket, GitLab, or AWS CodeCommit.
+- **[Break Your Monolith](/concepts/event-driven-architecture/)**: Create new services with a modern architecture or modernize your existing apps into a event-driven microservices architecture.
 
-## Get Help
+## Developer Self-Service
 
-- Join our [Discord community](https://amplication.com/discord) for support and collaboration.
-- Check out our [YouTube channel](https://www.youtube.com/c/Amplicationcom) for video tutorials.
-- Read our [blog](https://amplication.com/blog) for in-depth articles and updates.
+Enable developers to quickly create standardized services through our Service Catalog.
+Import existing databases, extend functionality through custom code, leverage multiple database options, and create custom plugins.
```

Comment:
1. Let's push the import to the end of the list here (and below)
2. Let's remove the "extend functionality through custom code" - here and below (as it adds nothing "real" to our capabilities list).
3. Instead of talking about creating custom plugins- let's talk about using plugins to add functionality to the services efficiently and in a standard way. This can be done using the community plugins or creating private plugins. We can mention here the categories of plugins like deployment, security etc.